openapi/gateway.json

```diff
@@ -1197,24 +1197,12 @@
         "type": "object",
         "properties": {
           "control_plane": {
-            "type": "array",
-            "items": {
-              "type": "integer",
-              "format": "uint8",
-              "minimum": 0
-            },
-            "minItems": 32,
-            "maxItems": 32
+            "type": "string",
+            "format": "hex string (32 bytes)"
           },
           "host_phase_2": {
-            "type": "array",
-            "items": {
-              "type": "integer",
-              "format": "uint8",
-              "minimum": 0
-            },
-            "minItems": 32,
-            "maxItems": 32
```

Comment:
This API changes in a way that I think makes sense: now you'll be able to read artifact hashes as hex strings in messages.